app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    content = command.split(" ")
```

Comment:
Consider checking if the length of `content` is 3 before accessing `content[1]` and `content[2]` to avoid potential index errors if the command is malformed.